gnd/src/main/java/com/google/android/gnd/ui/editobservation/MultipleChoiceFieldViewModel.java

```diff
@@ -37,4 +39,10 @@ public void onShowDialog() {
   MutableLiveData<Nil> getShowDialogClicks() {
     return showDialogClicks;
   }
+
+  Optional<MultipleChoiceResponse> getCurrentResponse() {
+    return getResponse().getValue() == null
```

Comment:
That results in `Optional<Optional< MultipleChoiceResponse>>` as `getResponse()` already returns optional. Couldn't think of any other way to do the check. Do you have an idea?

Reply:
If getValue() returns an Optional, it shouldn't also be nullable. Would it be difficult to update it so that getValue() is `@NonNull`?

Reply:
`getValue()` is a method of `LiveData` class. We can't update it.

Reply:
Ah, sorry, you're right. In general, moving forward we should probably replace LiveData<Optional<T>> with LiveData<T>; we'd like to avoid cases where a variable can be both null or Optional. Related to #679.